addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -594,6 +594,12 @@ body.editor_enable.editor_has_snippets {
     background-color: var(--o-we-snippets-bg-color);
     color: var(--o-we-snippets-color);
 
+    &.ui-draggable-dragging {
+        transform: rotate(3deg);
+        box-shadow: 0 5px 25px -10px black;
+        transition: transform 0.6s, box-shadow 0.3s;
```

Comment:
I tried with 0.3s as suggested by @qsm-odoo and I prefer :-)

Reply:
Ok, I update that right away